dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -1,19 +1,19 @@
 class Plc::EnrollmentEvaluationsController < ApplicationController
```

Comment:
Since CourseUnits now host the evaluation, everything in this file that was talking to courses now needs to talk to course_units